_drafts/2023-02-13-cheating-the-system-for-fun-and-profit.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+permalink: cheating-the-system-for-fun-and-profit
+title: 'Cheating the System for Fun and Profit'
+tags: [accessibility]
+---
+
+Okay, that title might just be a little bit of clickbait. The idea is to use
+macOS VoiceOver and Hover Text to navigate through Xcode's simulator, helping
+test your (iOS) app without having to take out your iOS device... mostly. But
```

Comment:
Personally I'd clarify a little up front that Mac VO and iOS VO are not the same so testing on device is still required, this is just a technique we can use to get feedback earlier.